src/lib.rs

```diff
@@ -262,6 +296,12 @@ fn macho_get_section<'a>(macho: &mach::MachO<'a>, section_name: &str) -> Option<
     None
 }
 
+fn macho_get_section_by_indx<'a>(macho: &mach::MachO<'a>, indx: usize) -> Option<Section<'a>> {
+    macho.segments.iter().flat_map(|x| x).flat_map(|x| x)
```

Comment:
Just so I don't have to figure this out... one of these `flat_map`s concatenates the sections? What's the other one for?